packages/edit-site/src/components/header/dashboard-button/style.scss

```diff
@@ -0,0 +1,42 @@
+.edit-site-dashboard-button_wrapper {
```

Comment:
Please note that we should use a ["loose BEM" naming](https://github.com/WordPress/gutenberg/blob/f485f1f6a5556893b7774f2e9ef3320b0884c620/docs/contributors/coding-guidelines.md#css).
Modifier should be preceded by a double underscore.
E.g. `. edit-site-dashboard-button__wrapper`

(There are a few other occurrences of this in the PR, but I'll leave you the _fun_ of finding them 😛 )

Reply:
I was confused by `FullscreenModeClose` because that component is not following the BEM. 🤔 Thanks for letting me know, fixed it up!